packages/text/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "version": "1.0.0",
+  "version": "1.0.1",
   "name": "@nodejs-loaders/text",
-  "description": "Text loader for Node.js",
+  "description": "Extend node to support plain-text files via customisation hooks.",
```

Comment:
```suggestion
  "description": "Extend node to support text files via customisation hooks.",
```

Reply:
All non-binary files are text-files; these are specifically files that should be treated as plain-text (mime-type `text/plain`).

Reply:
Okay